src/main/java/com/minecolonies/coremod/colony/Structures.java

```diff
@@ -115,18 +115,31 @@ public static void init()
      * Loads all styles saved in ["/assets/minecolonies/schematics/"].
      * Puts these in {@link #md5Map}, with key being the fullname of the structure (schematics/stone/Builder1).
      */
+    // The same exception will be triggered in the 2nd catch with logging this time.
+    @SuppressWarnings("squid:S1166")
     private static void loadStyleMaps()
```

Comment:
Agian change this to use the resource system to load a specific resource from the JAR.